### PR TITLE
[4486][3.1.x] RTE 5 toolbar disappears when anything is clicked

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/4486

### CODE REVIEW

File: ui/scss/forms-engine.scss [hunk 1]
```diff
@@ -116,6 +116,10 @@ div.xdsoft_datetimepicker {
   z-index: 1040 !important;
 }
 
+.tox-fullscreen .tox.tox-tinymce-aux, .tox-fullscreen~.tox.tox-tinymce-aux {
+  z-index: 1201 !important;
+}
```
Comment (rart) [inline ui/scss/forms-engine.scss]: What's this for? Why is this needed?

File: static-assets/components/cstudio-forms/controls/rte-tinymce5.js [hunk 2]
```diff
@@ -314,14 +314,17 @@ CStudioAuthoring.Module.requireModule(
           editor = tinymce.init({
             selector: '#' + rteId,
             width: _thisControl.rteWidth,
-            height: _thisControl.rteHeight,
+            // As of 3.1.14, the toolbar is moved to be part of the editor text field (not stuck/floating at the top of the window). 
+            // Adding 78px (toolbar's height) so that the toolbar doesn't eat up on the height set on the content modelling tool.
+            height: _thisControl.rteHeight + 78,
```
Comment (rart) [inline static-assets/components/cstudio-forms/controls/rte-tinymce5.js:319]: Use this comment on develop branch too.

File: ui/scss/forms-engine.scss (hunk 1 above)
Comment (jvega190) [reply ui/scss/forms-engine.scss]: While testing I found an issue in fullscreen mode where the menu bar items (when clicking on an entry) were not showing

Comment (rart) [reply ui/scss/forms-engine.scss]: **Why**? Instead of adding important to this one, should we be getting rid of other styles elsewhere?

**Review by rart — APPROVED** [review]

**Review by rart — APPROVED** [review]
